src/dotnet/APIView/APIViewWeb/Repositories/ReviewManager.cs

```diff
@@ -255,6 +255,7 @@ private void InitializeFromCodeFile(ReviewCodeFileModel file, CodeFile codeFile)
             file.Language = codeFile.Language;
             file.VersionString = codeFile.VersionString;
             file.Name = codeFile.Name;
```

Comment:
How is name different from the package name?

Reply:
Currently Name is package name and version name. Package Name is to standardize this to have only package name for lookup purpose when autodetecting API difference for a package is added.